libgalois/include/katana/analytics/BfsSsspImplementationBase.h

```diff
@@ -121,8 +121,9 @@ struct BfsSsspImplementationBase {
   template <typename WL, typename TileMaker>
   void PushEdgeTiles(
       WL& wl, katana::PropertyGraph* graph, GNode src, const TileMaker& f) {
-    auto beg = graph->edges(src).first;
-    const auto end = graph->edges(src).second;
+    auto edge_range = graph->topology().edge_range(src);
```

Comment:
Let's not move back to the topology API. I was hoping we could remove this API going forward. Same below.